coderd/database/dbauthz/dbauthz.go

```diff
@@ -5101,6 +5105,13 @@ func (q *querier) UpsertOAuthSigningKey(ctx context.Context, value string) error
 	return q.db.UpsertOAuthSigningKey(ctx, value)
 }
 
+func (q *querier) UpsertPrebuildsSettings(ctx context.Context, value string) error {
+	if err := q.authorizeContext(ctx, policy.ActionUpdate, rbac.ResourceDeploymentConfig); err != nil {
```

Comment:
Just to clarify here on the rbac side, this means that only a user with this permission can update the setting. From https://github.com/coder/coder/blob/main/coderd/rbac/roles.go currently, only the `auditorRole` has this permission. Is that intended?

Reply:
Auditor only has read though? https://github.com/coder/coder/blob/8a69f6af173d5d4a438289a98e69a1b61030b1ec/coderd/rbac/roles.go#L327